runners-msi/common/common-directories.wxi

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For these wxi files, it's almost impossible to see what has changed because the paths have changed as well. Could you put the paths at least temporarily back as they were and only change if necessary at the end of the review?

If that's too much work, just say so and I'll download and use some local tools to compare. However, it's always good to play to the lowest common denominator in viewing diffs, which is the github online view.